assets/src/functions/fetchJson.js

```diff
@@ -0,0 +1,4 @@
+export const fetchJson = async(url) => {
```

Comment:
Can you add an explanation for the existence of this function here (nonce cached problem) ? There's a [PR on gutenberg that seems to add an option `withNonce` in `apiFetch`](https://github.com/WordPress/gutenberg/pull/24623/files#diff-89239b7bd7836db63dd0ddb87c358ceb6fcfc8227469ffe796e85501338805d9R24), we might not need this anymore if it is merged at some point.

Reply:
`apiFetch` doesn't really do anything except adding the nonce (and allowing to use [some middleware we probably don't need](https://github.com/WordPress/gutenberg/blob/master/packages/api-fetch/README.md#built-in-middlewares)). The nonce makes no sense to use on a public endpoint, and we're starting to cache these endpoint responses in Cloudflare.

Reply:
And also using `apiFetch` means loading yet another resource before `frontendIndex.js`. But maybe I'm missing something about `apiFetch` that could still make it useful :slightly_smiling_face: